FiveCalls/FiveCalls/IssueDone.swift

```diff
@@ -115,7 +115,6 @@ struct IssueDone: View {
             }
             .padding(.horizontal)
         }
-        .navigationBarHidden(true)
```

Comment:
Let's keep this line, I don't think you should be able to go back from the done page

Reply:
Good catch! Updated